angular-reactive-validation/package.json

```diff
@@ -20,18 +20,18 @@
   "private": false,
   "dependencies": {},
   "peerDependencies": {
-    "@angular/core": "^11.0.0",
-    "@angular/common": "^11.0.0",
-    "@angular/forms": "^11.0.0",
-    "rxjs": "^6.5.4"
+    "@angular/core": "^12.2.15",
+    "@angular/common": "^12.2.15",
+    "@angular/forms": "^12.2.15",
+    "rxjs": "^6.6.7"
```

Comment:
**thought**: The package can be compatible with any Angular 12.x version by changing the `peerDependencies` to be the lowest possible version we want to be compatible with. In this case `"^12.0.0"` for Angular and `"^6.5.3"` for rxjs (so can actually be lower than the previous version set there, oops).

Reply:
done